### PR TITLE
add imputation warning

### LINKED ISSUE #267: Give user feedback how many values were imputed
Maybe even for every feature or so.
Some feedback would be useful.

### PR DESCRIPTION
**Pull request by Zethson**
Signed-off-by: zethson <lukas.heumos@posteo.net>

fixes #267

### CODE REVIEW

**Comment by Zethson** [comment]
@Imipenem only var for now and no obs warnings, but I think that's fine.

File: tests/api/preprocessing/test_data_imputation/test_impute_num.csv [hunk 1]
```diff
@@ -2,4 +2,4 @@ col1,col2,col3
 1,2,
 3,4,5
 ,6,7
-8,,10
```
Comment (Zethson) [inline tests/api/preprocessing/test_data_imputation/test_impute_num.csv:5]: Done to have one column with less than 25% missing values.

File: ehrapy/api/preprocessing/_data_imputation.py [hunk 2]
```diff
@@ -319,6 +319,35 @@ def miss_forest_impute(
     return adata
 
 
+def _warn_imputation_threshold(adata: AnnData, var_names: list[str] | None, threshold: int = 30) -> dict[str, int]:
```
Comment (Imipenem) [inline ehrapy/api/preprocessing/_data_imputation.py:322]: Where is this function acutally used?

**Comment by Zethson** [comment]
Lol I guess I forgot to add it to all imputation functions.  Saturday, 29 January 2022, 09:50AM +01:00 from Philipp Ehmele  ***@***.*** :

***@***.*** commented on this pull request.

**Review by Imipenem — APPROVED** [review]
LGTM

**Pull request merged**